ddtrace/contrib/flask/patch.py

```diff
@@ -121,6 +121,9 @@ def patch():
     _w("flask", "render_template", traced_render_template)
     _w("flask", "render_template_string", traced_render_template_string)
 
+    # trace stream_with_context functions
+    _w("flask.helpers", "stream_with_context", traced_stream_with_context)
+    _w("flask", "stream_with_context", traced_stream_with_context)
```

Comment:
do we need both?

Reply:
Yes because `stream_with_context` can be imported from both modules. Although it seems like there's only documentation for [flask.stream_with_context](https://flask.palletsprojects.com/en/2.1.x/patterns/streaming/#streaming-with-context) so we could just support that function import. I have no strong opinions here